shaderlib/project/project.glsl

```diff
@@ -226,6 +226,9 @@ vec4 project_position(vec4 position) {
 //
 
 vec4 project_to_clipspace(vec4 position) {
+  if (projectionMode == PROJECT_MERCATOR_OFFSETS) {
+    return projectionMatrix * vec4(position.xy, 0.0, 0.0) + projectionCenter;
```

Comment:
For a separate PR:

We should just be able to build this offset into the `projectionMatrix` in `Viewport.getUniforms`?  

Then we could also apply a model matrix in `getUniforms`. 

Maybe we can build in the scaling as well, and then both `project_scale` and `project_position` become identity functions under OFFSET_METERS mode?